pkg/api/resource/quantity_test.go

```diff
@@ -488,7 +488,9 @@ func TestUninitializedNoCrash(t *testing.T) {
 	q.Value()
 	q.MilliValue()
 	q.Copy()
-	q.String()
+	if str := q.String(); str != "0" {
```

Comment:
Since this test is really testing that it's not crashing, maybe `_ = q.String()` would be more appropriate.